backend/src/main/java/com/github/boltzmann/biokiste/backend/service/BoxDetailsService.java

```diff
@@ -30,4 +32,28 @@ public List<Item> getContentByBoxID(String boxId) {
                 .orElseThrow(() -> new NoSuchOrganicBoxException("There is no box with id " + boxId));
         return itemDetailsService.getItemsById(box.getContent());
     }
+
+    public OrganicBox addSubscriptionOfUserToBox(String userId, String boxId) {
+        OrganicBox box = organicBoxRepository.findById(boxId)
+                .orElseThrow(() -> new NoSuchOrganicBoxException("There is no box with id " + boxId));
+        List<String> oldBoxList = box.getCustomers();
+        if(oldBoxList != null) {
+            List<String> tmp = box.getCustomers();
```

Comment:
ist das hier nicht doppelt? Könntest du dir Zeile 41 nicht sparen und in 42 oldBoxList.add(userId)... setzen?